src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -150,7 +149,10 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
             objData.Alignment = objData.TargetPointerSize;
             objData.DefinedSymbols.Add(this);
 
-            ComputeOptionalEETypeFields(factory);
+            if (!(_type is CanonBaseType))
```

Comment:
Ah! that's the API I was looking for! I remember seeing it somewhere, but didn't know it was implemented as a static method on the context. I kept typing _type. and looking at the intellisense suggestions :)

---
In reply to: [97863824](https://github.com/dotnet/corert/pull/2575#discussion_r97863824) [](ancestors = 97863824)